Makefile

```
@@ -16,11 +16,11 @@ PREV_RELEASE=release-1.24
 # NB: If we lock images to commits/versions, this could affect the image
 # version matching in ./get-addon-templates. Be careful here, and verify
 # any images we need based on commit are matched/substituted correctly.
-CEPH_CSI_COMMIT=00f88e58a0b08be3d7ea009a0cfde8dca23021bf  # v3.5.1
-COREDNS_COMMIT=cb075b0a3810ccc0cb83339ddbce4b094c8f499b  # v1.9.0
-OPENSTACK_PROVIDER_COMMIT=1b5899e9f4c7c35e98e0be94732a88452b36cd15  # v1.23.1
-KUBE_DASHBOARD_COMMIT=52001304228c58430485674d21458527c8e6d86a  # v2.5.1
-KUBE_STATE_METRICS_COMMIT=ef8bede4064f04472a8c2fe8c281e6bf97de4329  # v2.4.2
+CEPH_CSI_COMMIT=47b59ee5a430f66a88913bea1a6ac1961c8ff552 # v3.7.2
+COREDNS_COMMIT=bfba43007a6afb04826911f9daeb5f303168fbf6 #v1.14.0
```

[This coredns commit](https://github.com/coredns/deployment/commit/bfba43007a6afb04826911f9daeb5f303168fbf6) dates from Nov 2020 and [comes with coredns 1.8.0](https://github.com/coredns/deployment/blob/bfba43007a6afb04826911f9daeb5f303168fbf6/kubernetes/coredns.yaml.sed#L112).

I see how you landed here, from that weird coredns-1.14.0 tag on the same repo. They're not actually tagging coredns releases on this repo though. The [latest release](https://github.com/coredns/coredns/releases) is actually 1.10.0.

I suggest using [31e9b6e2229300280f9788b1eaf1eb18c1b2d5c6](https://github.com/coredns/deployment/commit/31e9b6e2229300280f9788b1eaf1eb18c1b2d5c6) instead, which is the latest commit on the master branch of the deployment repo and [comes with coredns 1.9.4](https://github.com/coredns/deployment/blob/31e9b6e2229300280f9788b1eaf1eb18c1b2d5c6/kubernetes/coredns.yaml.sed#L120). That's not the latest release of coredns, but it's the latest we're going to get from the deployment repo that is now deprecated. Further updates will require us to pull coredns templates from a different source.

```suggestion
COREDNS_COMMIT=31e9b6e2229300280f9788b1eaf1eb18c1b2d5c6 #v1.9.4
```